presto-product-tests/src/main/java/io/prestosql/tests/hive/TestGrantRevoke.java

```diff
@@ -231,6 +232,20 @@ public void testViewOwnerPrivileges()
         });
     }
 
+    @Test(groups = {AUTHORIZATION, PROFILE_SPECIFIC_TESTS})
+    public void testTablePrivilegesWithHiveOnlyViews()
+    {
+        executeWith(createViewAs("hive_only_view", format("SELECT * FROM %s", tableName), onHive()), view -> {
+            assertThatThrownBy(() -> onPresto().executeQuery(format("SELECT * FROM %s", view.getName())))
+                    .hasMessageContaining("Hive views are not supported");
+
+            assertThat(onPresto().executeQuery("SELECT DISTINCT table_name FROM information_schema.table_privileges"))
```

Comment:
can you do `WHERE table_name LIKE '%hive_only_view%'` and assert the row count is 0?

Reply:
then we won't be testing that `table_privileges` still returns _something_.
it could as well return nothing (think: catch applied in a wrong place)